packages/opentelemetry-nodejs/README.md

```diff
@@ -0,0 +1,78 @@
+# SuperTokens Plugin Captcha
+
+Add OpenTelemetry logging to all SuperTokens APIs and function calls.
+
+## Installation
+
+```bash
+npm install @supertokens-plugins/opentelemetry-nodejs
+```
+
+## Quick Start
+
+Initialize the plugin in your SuperTokens backend configuration:
+
+```typescript
+import SuperTokens from "supertokens-node";
+import OpenTelemetryPlugin from "@supertokens-plugins/opentelemetry-nodejs";
+
+SuperTokens.init({
+  supertokens: {
+    connectionURI: "...",
+  },
+  appInfo: {
+    // your app info
+  },
+  recipeList: [
+    // your recipes
+  ],
+  plugins: [
+    OpenTelemetryPlugin.init(),
+  ],
+});
+```
+
+> [!IMPORTANT]  
+> You also have to install and configure the OpenTelemetry SDK. See here for more details: https://opentelemetry.io/docs/languages/js/getting-started/nodejs/#instrumentation
+
+## Traces added
+
+This plugin manually adds traces to all overrideable functions and APIs. By initializing the OpenTelemetry SDK, you automatically get API level. Debug logs are currently not part of the logged events.
+
+## Customization
+
+### Removing sensitive data
+
+By default the plugin removes the following fields from traces: "password", "email", "phoneNumber", "email", "emails", "phoneNumbers", "accessToken", "refreshToken"
+You can add (or remove) items from this list by overriding the `getSensitiveFields` function:
+
+
+```typescript
+import SuperTokens from "supertokens-node";
+import OpenTelemetryPlugin from "@supertokens-plugins/opentelemetry-nodejs";
+
+SuperTokens.init({
+  supertokens: {
+    connectionURI: "...",
+  },
+  appInfo: {
+    // your app info
+  },
+  recipeList: [
+    // your recipes
+  ],
+  plugins: [
+    OpenTelemetryPlugin.init({
+      override: () => ({
+        getSensitiveFields: (defaultSensitiveFields: string[]) => [...defaultSensitiveFields, "randomField"],
```

Comment:
Do you need to import `defaultSensitiveFields` from somewhere or should it be initiated in the snippet?